### DISCUSSION TITLE
How to add quad to sprite?

### DISCUSSION THREAD

**Discussion by petropavliuk-valtech** [replit/kaboom#527] [Q&A] [answered]
I tried to use the quad with the sprite() but its failing with the error _**o.scale is not a function**_

**Comment by slmjkdbtl** [comment, marked as answer]
That's a bug. Fixed in v2000.2.7!

**Reply by petropavliuk-valtech** [reply]
@slmjkdbtl , works perfect! Thanks a lot!
